### PR TITLE
chore/build: use own fork of sodiumoxide

### PR DESCRIPTION
**Pull request by Fraser999**
This change is [Reviewable](https://reviewable.io/reviews/maidsafe/secure_serialisation/18)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @dirvine

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+reorder_imports = true
```
Comment (dirvine) [inline rustfmt.toml]: Did you not just remove this file from DataChains ?

Comment (ustulation) [inline rustfmt.toml]: he's biased it seems :P ..
It seems OK though, no ? It will make things uniform.

Comment (Fraser999) [inline rustfmt.toml]: No - I removed the clippy toml since clippy was passing there without any exclusions being required :)

Comment (dirvine) [inline rustfmt.toml]: Ah cool, sorry for the noise. I agree with this file being in place though

**Comment by dirvine** [comment]
:lgtm:

---

Reviewed 3 of 4 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/secure_serialisation/18#-:-KPIvMa0HdshPnkVhDJR:bnfp4nl)*

**Pull request merged**